core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -84,27 +84,13 @@
   /**
    * Creates a new file output stream.
    *
-   * @param context the file system context
-   * @param path the file path
-   * @param options the client options
-   * @throws IOException if an I/O error occurs
-   */
-  public FileOutStream(FileSystemContext context, AlluxioURI path, OutStreamOptions options)
-      throws IOException {
-    this(path, options, context, UnderFileSystemFileOutStream.Factory.get());
-  }
```

Comment:
@peisun1115 @jsimsa Do you know if this constructor was kept for backward compatibility? IMO the `FileOutStream` constructor should not be public facing since users should only get them from invoking the `FileSystem` object.